databases/week3/hyf_week3_homework.sql

```diff
@@ -0,0 +1,165 @@
+VALUES (6, 4, '2021-08-12 00:00:00', '55 55 55 55', 'John', 'zxc@mail.com');
+
+INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES (2, 5, '2021-08-10 00:00:00', '66 66 66 66', 'Angel', 'vbn@mail.com');
+
+INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES (7, 6, '2021-08-11 00:00:00', '77 77 77 77', 'Don', 'qwe@mail.com');
+
+INSERT INTO review (title, description, meal_id, stars, created_date)
+VALUES ('Very Delicious', 'Best food of my life', 1, 5 , '2021-08-11 00:00:00');
+
+INSERT INTO review (title, description, meal_id, stars, created_date)
+VALUES ('Amazing', 'Amazing Food', 2, 5 , '2021-08-11 00:00:00');
+
+INSERT INTO review (title, description, meal_id, stars, created_date)
+VALUES ('Awesome', 'Awesome Food', 4, 5 , '2021-08-12 00:00:00');
+
+INSERT INTO review (title, description, meal_id, stars, created_date)
+VALUES ('Really Bad', 'Bad taste and very salty', 5, 1 , '2021-08-11 00:00:00');
+
+INSERT INTO review (title, description, meal_id, stars, created_date)
+VALUES ('Good', 'Normal Taste', 6, 3 , '2021-08-11 00:00:00');
+
+-- QUERIES
+-- Get all meals
+SELECT *
+FROM meal;
+
+-- Add a new review
+INSERT INTO review (title, description, meal_id, stars, created_date)
+VALUES ('A little bit salty', 'Food was little bit salty but it was delicious', 3, 4 , '2021-08-11 00:00:00');
+
+-- Get a review with any id, fx 1
+
+SELECT *
+FROM review
+WHERE meal_id = 2;
+
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+
+UPDATE review
+SET description = 'Very very delicious'
+WHERE review.id = 1;
+
+-- Delete a review with any id, fx 1
+
+DELETE FROM review WHERE id = 1;
+
+-- Get meals that has a price smaller than a specific price fx 90
+
+SELECT *
+FROM meal
+WHERE meal.price < 360;
+
+-- Get meals that still has available reservations
+
+SELECT *
+FROM meal
+JOIN reservation ON meal.id = reservation.meal_id
+WHERE reservation.number_of_guests < meal.max_reservations;
+
+-- Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+
+SELECT *
+FROM meal
+WHERE meal.title like '%yaprak%';
+
+-- Get meals that has been created between two dates
+
+SELECT *
+FROM meal
+WHERE meal.created_date < '2021-08-12'
+AND meal.created_date > '2021-08-09';
+
+-- Get only specific number of meals fx return only 5 meals
+
+SELECT *
+FROM meal
+LIMIT 2;
+
+-- Get the meals that have good reviews
+
+SELECT meal.* , review.stars
+FROM meal
+JOIN review ON meal.id = review.meal_id
+WHERE review.stars > 3;
+
+-- Get reservations for a specific meal sorted by created_date
+
+SELECT *
+FROM reservation
+JOIN meal ON meal.id = reservation.meal_id
+WHERE meal.title LIKE '%yaprak%'
+ORDER BY reservation.created_date ASC;
+
+-- Sort all meals by average number of stars in the reviews
```

Comment:
Nitpicking, but put your comment directly above the query it describes, it gives a stronger visual cue as to where it belongs